libs/sdk-ui-kit/src/Overlay/typings.ts

```diff
@@ -0,0 +1,62 @@
+// (C) 2020 GoodData Corporation
+import { IAlignPoint } from "../typings/positioning";
+import { Alignment } from "../typings/overlay";
+
+/**
+ * @public
```

Comment:
This comment has no effect, please remove it

Reply:
Removed.